extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxRenderServiceHelper.php

```diff
@@ -214,6 +214,27 @@ private function getThumbnail( $title ) {
 	}
 
 	/**
+	 * @desc return physicalsize thumbnail URL
+	 * @todo Special case for starwars.wikia.com,
+	 * @todo rethink approach to images as a part of https://wikia-inc.atlassian.net/browse/DAT-3075
+	 * @param $thumbnail \MediaTransformOutput
+	 * @return string thumbnail URL
+	 */
+	private function getPhysicalSizeThumbUrl( $thumbnail ) {
+		global $wgPortableInfoboxCustomImageWidth;
+
+		if ( !empty( $wgPortableInfoboxCustomImageWidth ) ) {
+			$file = $thumbnail->file;
+			$height = min( self::MAX_DESKTOP_THUMBNAIL_HEIGHT, $file->getHeight() );
```

Comment:
I wonder whether we want to use self::MAX_DESKTOP_THUMBNAIL_HEIGHT or rather a proportionally scaled value to match custom width passed through the WF var